src/lib/pages/types.ts

```diff
@@ -57,14 +57,26 @@ export namespace V2 {
   }
 
   export interface Page {
+    uuid?: string,
```

Comment:
I think I'd leave the uuid out of this base type. Then extend it for other types that need it.

Reply:
the uuid is, for some reason, in the schema of every request https://admin-master.dud4.co/admin/api-docs#/Multiscreen%20Sites%20Pages